core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -383,6 +392,36 @@ var TopicsAndSkillsDashboardPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
+  this.unassignSkillWithIndex = async function(skillDescription, topicIndex) {
+    await waitFor.visibilityOf(skillsTable,
+      'Skill table taking too long to appear.');
+    var skillIndex = -1;
+    for (var i = 0; i < skillDescriptions.length; i++) {
+      var skillDescriptionText = (
+        await (await skillDescriptions.get(i)).getText());
+      if (skillDescriptionText === skillDescription) {
+        skillIndex = i;
+        break;
+      }
+    }
+    var skillEditOptionBox = skillEditOptions.get(skillIndex);
```

Comment:
Could you assert/expect that skillIndex != -1? That'd be more clear than an index out of bounds exception

Reply:
Yeah right.
Done.